File: ERPclient/ERPclient/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ERPclient
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            // TODO: 这行代码将数据加载到表“liledsDataSet.tb_proinfo”中。您可以根据需要移动或移除它。
            this.tb_proinfoTableAdapter.Fill(this.liledsDataSet.tb_proinfo);

        }
    }
}

File: inventorysys.asp/inventorysys.asp/Form1.vb
Public Class Form1

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        'TODO: 这行代码将数据加载到表“LiledsDataSet.tb_proinfo”中。您可以根据需要移动或移除它。
        Me.Tb_proinfoTableAdapter.Fill(Me.LiledsDataSet.tb_proinfo)

    End Sub

    Private Sub DataGridView1_CellContentClick(ByVal sender As System.Object, ByVal e As System.Windows.Forms.DataGridViewCellEventArgs) Handles DataGridView1.CellContentClick

    End Sub

    Private Sub FillToolStripButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles FillToolStripButton.Click
        Try
            Me.Tb_proinfoTableAdapter.Fill(Me.LiledsDataSet.tb_proinfo)
        Catch ex As System.Exception
            System.Windows.Forms.MessageBox.Show(ex.Message)
        End Try

    End Sub
End Class
